scripts/build/TestAssets.props

```diff
@@ -3,7 +3,6 @@
     <PropertyGroup>
         <!-- Not able to add TargetFrameworks here due to: https://github.com/dotnet/sdk/issues/1110 -->
         <NetCoreAppTargetFrameWork Condition="'$(TargetFramework)' != '' AND $(TargetFramework.StartsWith('netcoreapp'))">true</NetCoreAppTargetFrameWork>
-	    <PackageTargetFallback Condition="$(NetCoreAppTargetFrameWork) == 'true' ">$(PackageTargetFallback);dnxcore50;portable-net45+win8</PackageTargetFallback>
```

Comment:
It appears all of our dependent packages have built for netstandard1.5 and netcoreapp1.0. We no more need `PackageTargetFallback`, unless we add a new dependency which didn't build for netstandard1.5 or netcoreapp1.0.